PL2/PL2-shared/src/main/java/nl/tudelft/pl2016gr2/model/OriginalGraph.java

```diff
@@ -17,10 +17,14 @@ public OriginalGraph() {
   /**
    * Print a string representation of this graph.
    */
-  public void print() {
+  @Override
+  public String toString() {
+    StringBuilder sb = new StringBuilder();
     for (Node node : nodes.values()) {
-      System.out.println(node);
+      sb.append(node);
+      sb.append("\n");
```

Comment:
My opinion is that chaining actually makes it less readable. As \n creates a new line, I also put it on a new line.